### ISSUE TITLE
Accessibility issues: Keyboard cannot focus on and control the eye and dice icon of the password field in entry window

### ISSUE THREAD

**Issue by LittleStar-VIP** [keepassxreboot/keepassxc#9059]
Hello,
This is a screen reader user, who rely on screen reader to navigate among the application gui elements, and I found that I have problem in interacting with some of the elements

## Steps to Reproduce
[NOTE]: # ( Provide a simple set of steps to reproduce this bug. )
1. create a new entry
2. use keyboard tab key to tab to the pass word field
3. according to the user guide, there should be an eye icon to visualize password, and a dice icon to generate password, but so far, I am not able to click onto these icons using screen reader

## Expected Behavior
[NOTE]: # ( Tell us what you expected to happen )
the eye and dice icon should receive keyboard focus and should be operable by keyboard control

## Actual Behavior
[NOTE]: # ( Tell us what actually happens )
The screen reader only report to me that there is a password field, and nothing else can be found

## Context
[NOTE]: # ( Give us any additional information you may have. )
The screen reader that I am using is NVDA 2022.4

[NOTE]: # ( Paste debug info from Help → About here )
KeePassXC - VERSION
KeePassXC 2.7.4

[NOTE]: # ( Pick choices based on your environment )
Operating System: Windows
Desktop Env: not sure what is this
Windowing System: Windows 10 Home ed

**Comment by droidmonkey** [comment]
Use Ctrl + H to toggle password visibility, use Ctrl + G to open the generator. They are not focusable buttons but embedded buttons in the text line. 

We'll add a description to the text reading that mentions the shortcuts.

**Comment by michaelk83** [comment]
@droidmonkey I imagine that multi-key shortcuts are difficult for some users. Could these buttons be made focusable?

Alternatively, could these functions be placed in a context menu?

(edit: I suppose there's the sticky keys tool for such users, but having other single-key options to activate these functions would still be helpful.)

**Comment by droidmonkey** [comment]
I tried and it is not possible to make inline actions focusable with Qt (which is what these buttons are). A context menu is even less discoverable than these buttons and more code for no gain.
